engine/engine/content/builtins/manifests/web/engine_template.html

```diff
@@ -131,6 +133,12 @@
 		var game_canvas = document.getElementById('canvas');
 		var innerWidth = window.innerWidth;
 		var innerHeight = window.innerHeight - buttonHeight;
+		if (prevInnerWidth == innerWidth && prevInnerHeight == innerHeight)
+		{
+			return;
```

Comment:
we need this `return` to prevent flickering in a case when nothing changed and user just tap to the other app and went back.